specs/select.spec.md

```diff
@@ -0,0 +1,160 @@
+# Select + Select-Option
+
+## Overview
+
+The `select` is a component that provides a list of options for the user to select from.
+
+### Background
+
+There is a standard select html control, `<select>` and `<option>`, but it doesn't support the full level of detail, styling, etc that most design systems need or require. For context please see [MDN Docs](https://developer.mozilla.org/en-US/docs/web/html/element/select).
+
+### Use Cases
+
+- *A customer using the component on a web page.*
+On a web page a customer is shopping for a new shirt in size medium. The customer opens the size `select` and sees a list of `<option>`s and selects size medium.
+
+### Features
+- form association
+- focus delegation
+
+### Features
+- Select one or multiple `<fast-option/>`(s) using the `multiple` attribute
+- When Dropdown does not have enough screen real estate to open below `<fast-select>` it will open above.
+
+### Prior Art/Examples
+- [FAST Select (React)](https://www.npmjs.com/package/@microsoft/fast-components-react-msft)
+- [Ant Design](https://ant.design/components/select/)
+- [Carbon Design](https://www.carbondesignsystem.com/components/select/code/)
+- [Atlassian UI](https://atlaskit.atlassian.com/packages/core/select)
+- [Lightning Design System](https://www.lightningdesignsystem.com/components/select/)
+- [W3 Example](https://www.w3.org/TR/wai-aria-practices-1.1/examples/listbox/listbox-collapsible.html)
+
+---
+
+### API
+
+*Component Name*:
+- `fast-select`
+
+*Attributes:*
+- `auto-focus` - Automatically focuses the control
+- `disabled` - Disables the control
+- `menu-open` -If the menu of the `<fast-option>` is open or not
+- `multiple` - Allows user to select more than one `<fast-option>`
+- `name` - Name of the control
+- `options` - Returns an array of the `<fast-option>` elements contained by the `<fast-select>` element
+- `required` - Boolean value that sets the field as required
+- `selected-options` - An array of the selected `<fast-option>` elements
+- `value` - The value of select is typed as `any`
+
+*Events*
+- `change` - raise the change event for external parties to be informed of the `selected-option`'s value change.
+
+*Slots*
```

Comment:
Is this list of slots up to date per the latest open UI spec? I recall there was a listbox slot, for example.